nix/packages/make-dhcpd/make-dhcpd.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/evn bash
```

Comment:
nit: I don't think we need / want the shebang here if we are using the `makeShellApplication` function.

https://github.com/NixOS/nixpkgs/blob/a308483c4e3b6bf0112ea1420b30198396ac1d71/pkgs/build-support/trivial-builders/default.nix#L201-L206

Reply:
Good catch, thanks! I was somewhat trying to support some use outside of nix but looking at the file in the store:

```
#!/nix/store/516kai7nl5dxr792c0nzq0jp8m4zvxpi-bash-5.2p32/bin/bash


export PATH="/nix/store/13rjyks12ca0k5hr7lwpbvwdzj1rwkkz-dnsmasq-2.90/bin:$PATH"

#!/usr/bin/evn bash
...
```

Definitely dont want that

Reply:
fixed up